presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -150,12 +150,6 @@ protected AbstractTestQueries(QueryRunnerSupplier supplier)
         super(supplier);
     }
 
-    @Test
```

Comment:
This test isn't redundant. It's testing that we fail properly when we execute a query that has a parse error.